ThunderTable/InputInlineDatePickerViewCell.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  InputInlineDatePickerViewCell.swift
+//  ThunderTable
+//
+//  Created by Simon Mitchell on 16/09/2020.
+//  Copyright © 2020 3SidedCube. All rights reserved.
+//
+
+import UIKit
+
+open class InputInlineDatePickerViewCell: TableViewCell, DatePickerCell {
```

Comment:
More for understanding, what's special about iOS 14 here?

Reply:
@BenCShutt was this comment meant to be somewhere else? :D